package.json

```diff
@@ -71,6 +71,7 @@
     "./dist/h3-js.es.js": "./dist/browser/h3-js.es.js"
   },
   "devDependencies": {
+    "@types/markdown-it": "14.1.2",
```

Comment:
I had to force this to the latest version to fix the test suite, because typescript was failing to compile after doing a `yarn upgrade`. I'm not sure why yarn didn't upgrade this dependency automatically.